docs/source/conf.py

```diff
@@ -264,3 +266,54 @@ def setup(app):
     # copy changelog into source folder for documentation
     dest = osp.join(HERE, "./getting_started/changelog.md")
     shutil.copy(osp.join(HERE, "..", "..", "CHANGELOG.md"), dest)
+
+    # create rst files for all examples
+    # check what examples exist
+    examples_folder = osp.join(HERE, "..", "..", "ema_workbench", "examples")
+    examples = glob.glob(examples_folder + "/*.py")
+    ignore_list = {"__init__", "test_examples", "model_debugger"}
+
+    # get all existing rst files
+    rst_files = glob.glob(os.path.join(HERE, "examples", "*.rst"))
+    rst_files = {os.path.basename(os.path.normpath(entry)) for entry in rst_files}
+
+    # check which rst files exist
+    # TODO:: consider stripping out top level docstring and add this as normal text to example
+    # TODO then only include remaining lines.
+    with open(os.path.join(HERE, "example_template.txt")) as fh:
+        template = string.Template(fh.read())
+
+    # TODO:: at the moment no idea what happens if example is updated. Does this trigger a rebuild of the html page?
+    for example in examples:
+        base_name = os.path.basename(os.path.normpath(example))
+        base, ext = os.path.splitext(base_name)
+        if base in ignore_list:
+            continue
+
+        short_py_filename = f"{base}.py"
+        short_rst_filename = f"{base}.rst"
+        headerline = "=" * len(short_py_filename)
+
+        if short_rst_filename not in rst_files:
+            with open(os.path.join(HERE, "examples", short_rst_filename), "w") as fh:
+                relative_path = "../../ema_workbench/examples"
+                content = template.substitute(
+                    dict(short_filename=short_py_filename, headerline=headerline)
+                )
+                fh.write(content)
+        else:
+            rst_files.remove(short_rst_filename)
+
+    # these rst files are outdated because the example has been removed
+    for entry in rst_files:
+        fn = os.path.join(HERE, "examples", entry)
+        os.remove(fn)
+
+    notebooks = glob.glob(examples_folder + "/*.ipynb")
```

Comment:
You're already collecting notebooks here separately. Isn't it possible to put them under an separate header?